<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Prb</class>
 <widget class="QWidget" name="Prb">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pRasterBlaster</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>721</width>
     <height>631</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="inputFileNameLabel">
        <property name="text">
         <string>Input File Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputFileNameLineEdit">
        <property name="text">
         <string>sample-data/holdnorm_geographic_30min.tif</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="inputFileNameButton">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="inputPreviewButton">
        <property name="text">
         <string>View</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="outputFileNameLabel">
        <property name="text">
         <string>Output File Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="outputFileNameLineEdit">
        <property name="text">
         <string>sample-data/projected.tif</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="outputFileNameButton">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="outputPreviewButton">
        <property name="text">
         <string>View</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="resamplerLabel">
        <property name="text">
         <string>Resampler:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="resamplerComboBox">
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="srsLabel">
        <property name="text">
         <string>SRS:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="srsComboBox">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="elipsoidALabel">
        <property name="text">
         <string>Elipsoid A:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="elipsoidAComboBox">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="elipsoidBLabel">
        <property name="text">
         <string>Elipsoid B:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="elipsoidBComboBox">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="partitionSizeLabel">
          <property name="text">
           <string>Partition Size:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="partitionSizeLineEdit">
          <property name="text">
           <string>21600</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="fillValueLabel">
          <property name="text">
           <string>Fill Value:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="fillValueLineEdit">
          <property name="text">
           <string>32</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="reprojectButton">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Reproject</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="outputTextEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
